Lib/string.py

```diff
@@ -52,6 +52,8 @@ def capwords(s, sep=None):
 import re as _re
 from collections import ChainMap as _ChainMap
 
+_sentinel_dict = {}
```

Comment:
Because semantically the default value for the *mapping* parameter is an empty dict. The check `mapping is _sentinel_dict` is merely an optimization (we can avoid to create a ChainMap in this case).